playterminal/settings/base.py

```diff
@@ -101,3 +101,10 @@
 # https://docs.djangoproject.com/en/1.8/howto/static-files/
 
 STATIC_URL = '/static/'
+
+# Try to get the secrets values from the environment
+secret_vars = ['SECRET_KEY', 'STEPIC_CLIENT_ID', 'STEPIC_CLIENT_SECRET']
+for var in secret_vars:
+    value = os.getenv(var)
```

Comment:
Environmet variables is not the best place to store your secrets. Random article from google: http://movingfast.io/articles/environment-variables-considered-harmful/
I also use this approach somewhere, (e.g. https://github.com/bioinf/rnr-mal/blob/master/mal/settings/docker.py). I found it very convenient when you use docker for deployment. Still it doesn't look secure enough.
I'd prefer to use envvars only for CI and other staging environments, but on production use the ordinary settings file.

It seems that you share the same view, don't you?